source/developers/cook-books/how-tos/working-with-crafter-studios-api.rst

```diff
@@ -10,71 +10,113 @@ Working with Crafter Studio's APIs
 
 In this section, we'll show the basics of interacting with Crafter Studio APIs by performing the following:
 
-#. Authenticate
+#. Authentication
 #. Get a list of projects under management
 #. Write content to a project
 
 We’ll use CURL, a ubiquitous Linux command tool as our client.
 
 You can find the full Crafter Studio API for Crafter CMS :ref:`here<crafter-studio-api>`
 
-Most Crafter Studio API requires users to authenticate first.  When a user authenticates via the login API call, this establishes a session which is provided per J2EE in a **JSESSIONID** cookie.  Future calls must send this cookie.
-
-Crafter Studio APIs are protected using a CSRF Token so all ``POST``, ``PUT`` and ``DELETE`` calls need to include a cookie named **XSRF-TOKEN** and a header named **X-XSRF-TOKEN** with a matching value to work.
+Let's begin:
 
-As we'll see below, the Login API sends a header and a cookie CSRF/XSRF token value which must also be sent in subsequent calls. The value is arbitrary and is dictated by the caller. Strong values are more secure than predictable weak ones.
+#. **Authentication**
 
-Let's begin:
+   There are two authentication methods that can be used: JWT and HTTP Basic
 
-#. **Authenticate**
+   JWT is standard widely adopted as a secure authentication method, this is the prefered way. HTTP Basic is
+   simple but not as secure as JWT and for that reason it is disabled by default, however, it can be very useful for
+   local development environments.
```

Comment:
For the above paragraph, a typo and maybe reword to something like this:

```
JWT is a widely adopted standard for secure authentication and is the preferred way.  

HTTP Basic is simple but not as secure as JWT and for that reason it is disabled by default.  However, it can be very useful for local development environments
```